services-custom/s3-transfer-manager/src/main/java/software/amazon/awssdk/transfer/s3/internal/DownloadDirectoryHelper.java

```diff
@@ -148,7 +148,7 @@ private Path determineDestinationPath(DownloadDirectoryRequest downloadDirectory
                                           S3Object s3Object) {
         FileSystem fileSystem = downloadDirectoryRequest.destination().getFileSystem();
         String delimiter = listRequest.delimiter() == null ? DEFAULT_DELIMITER : listRequest.delimiter();
```

Comment:
You can remove this line, it's superfluous

Reply:
the delimiter is also used for the call to `getRelativePath`

Reply:
Ah, didn't see that